========================================================================================== FAILURES ==========================================================================================
_____________________________________________________________________________________ test_get_foo[foo] ______________________________________________________________________________________

input = 'foo'

    @pytest.mark.parametrize('input', [
        ('foo')
    ])
    def test_get_foo(input):
        output = foo()
>       assert(output == input)
E       AssertionError: assert 'bar' == 'foo'
E         
E         - foo
E         + bar

input      = 'foo'
output     = 'bar'

tests/test_foo.py:10: AssertionError
================================================================================== short test summary info ===================================================================================
FAILED tests/test_foo.py::test_get_foo[foo] - AssertionError: assert 'bar' == 'foo'
===================================================================================== 1 failed in 0.04s ======================================================================================